account_cutoff_base/models/account_cutoff_line.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2013-2021 Akretion (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountCutoffLine(models.Model):
+    _name = "account.cutoff.line"
+    _inherit = "analytic.mixin"
+    _description = "Account Cut-off Line"
+
+    parent_id = fields.Many2one("account.cutoff", string="Cut-off", ondelete="cascade")
+    name = fields.Char("Description")
+    company_currency_id = fields.Many2one(
+        related="parent_id.company_currency_id",
+        string="Company Currency",
+        readonly=True,
+    )
+    partner_id = fields.Many2one("res.partner", string="Partner", readonly=True)
+    quantity = fields.Float(digits="Product Unit of Measure", readonly=True)
+    price_unit = fields.Float(
```

Comment:
This could be a Monetary field

Reply:
No. price_unit are not monetary fields for decimal precision reasons, cf https://github.com/odoo/odoo/blob/16.0/addons/sale/models/sale_order_line.py#L139
A price_unit can have a decimal precision higher than the currency precision.